### DISCUSSION TITLE
How to query the Measurement result

### DISCUSSION THREAD

**Discussion by KanPlus** [keylime/keylime#1690] [General]
1 How to query   the  Measurement result of the PCR and IMA?
2 How to query   the  value of PCR and IMA in Measurement result?

**Comment by ansasaki** [comment]
Hi @KanPlus, could you please explain what are you trying to do? What exactly is you use case?

It would be nice to understand what you are trying to achieve, which data you need, and why.

**Comment by KanPlus** [comment]
Thanks, the use case is that monitor some VMs with keylime，and tried to report the monitor status with UI.

**Reply by ansasaki** [reply]
Please, provide more context. From what you are saying, can I assume you are trying to create a dashboard showing the attestation status for the VMs? What do you want to show on your dashboard?

**Reply by KanPlus** [reply]
Yes, the attestation status for the VMs tried to showed on the  dashboard with UI, the showed informations were as bellow:
1 The compare result and time of PCRs and IMA.
2 The  actual and baseline values of PCRs and IMA.
3 The reponse content of  PCRs and IMA.
